apis/python/tests/test_basic_anndata_io.py

```diff
@@ -149,24 +149,34 @@ def test_import_anndata(adata, ingest_modes, X_kind):
         obsm = exp.ms["RNA"].obsm
         assert sorted(obsm.keys()) == sorted(orig.obsm.keys())
         for key in list(orig.obsm.keys()):
-            assert obsm[key].metadata.get(metakey) == "SOMADenseNDArray"
+            assert (
+                obsm[key].metadata.get(metakey) == "SOMASparseNDArray"
```

Comment:
`in ("SOMASparseNDArray", "SOMADenseNDArray")`.

Also these blocks would be easier to understand if you pulled out the `obsm[key]` (and `varm[key]`) into a variable so that it's easier to follow that particular object.